src/ProjectNeon/Assets/Data/Enemies/AI/NoAI.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class NoAI : TurnAI
+{
+    public override PlayedCard Play()
+    {
+        throw new System.NotImplementedException();

```

Comment:
Never, ever throw an exception in Game Development. 

Logging errors can be allowed. But no exceptions, ever.